Assets/Scripts/Screeps3D/RoomObjects/UnpackUtility.cs

```diff
@@ -291,13 +291,20 @@ internal static void Store(IStoreObject obj, JSONObject data)
         internal static void ActionLog(IActionObject actionObject, JSONObject data)
         {
             var actionLog = data["actionLog"];
+            var creep = actionObject as Creep;
```

Comment:
no reason for this, was part of debug :)

Reply:
will remove